set.mm

```diff
@@ -214980,7 +214990,7 @@ is an important property of monoids (see ~ mndid ), and therefore also for
 
 $(
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-  Ordered sums in a magma
+  Group sum operation in magmas
```

Comment:
In the meantime, can you keep the existing titles ("Ordered sums in a magma") ?  (group sum in magmas sounds really weird)

Reply:
Since neither "ordered sum" nor "iterated sum" is defined yet, I would propose the titles "Iterated sums in a magma" and "Iterated sums in a monoid". This would be a first step into the direction of the new naming. 

And I would slightly enhance the comment of df-gsum accordingly: 

> Define the group sum **(also called "iterated sum")** for the structure ` G ` of a finite sequence

I want to keep the term "sum" for the moment because of the label/symbol `gsum`.

@benjub do you agree?

Reply:
I agree for this PR. For a long term solution, there is #3726.